feg/gateway/services/s6a_proxy/servicers/ul.go

```diff
@@ -55,6 +64,27 @@ func (s *s6aProxy) sendULR(sid string, req *protos.UpdateLocationRequest, retryC
 	return err
 }
 
+// createULR_Flags creates ULR Flags based on TS 29.272
+func createULR_Flags(req *protos.UpdateLocationRequest) int {
+	// ULR_FLAGS contains defaults flags for ULR-Flags
+	ulrFlags := ULR_FLAGS
+	if req.DualRegistration_5GIndicator {
```

Comment:
you can use req.GetDualRegistration_5GIndicator() instead, then you don't have to check for req == nil